craft_application/services/remotebuild.py

```diff
@@ -185,7 +184,7 @@ def fetch_logs(self, output_dir: pathlib.Path) -> Mapping[str, pathlib.Path | No
             if not url:
                 logs[build.arch_tag] = None
                 continue
-            filename = f"{project_name}_{build.arch_tag}-{fetch_time}.txt"
+            filename = f"{self._name}_{build.arch_tag}_{fetch_time}.txt"
```

Comment:
Changing delimiter before `fecht_time` seems like incompatible change for me.
Is it required?

Reply:
Since remote-build is still marked as experimental I think we're fine to do this.